src/main/java/util/Utility.java

```diff
@@ -90,6 +90,20 @@ public static boolean writeFile(String fileName, String content, int issueCount)
         return false;
     }
 
+    public static boolean deleteFile(String fileName) {
+        boolean validPath = !(fileName == null || fileName.isEmpty());
+        if (validPath) {
+            try {
+                Path repoFile = Paths.get(fileName);
+                Files.delete(repoFile);
+            } catch (IOException e) {
+                logger.error(e.getLocalizedMessage(), e);
+                return true;
+            }
+        }
+        return false;
```

Comment:
What does the return value of this signify? Might need javadocs to explain. If it's whether the file ultimately *does not* exist, there might be a missing return (in the `try` block).